cmd/aergocli/cmd/root.go

```diff
@@ -82,6 +82,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&keyFile, "tlskey", "", "client key file for TLS ")
 	rootCmd.PersistentFlags().StringVarP(&host, "host", "H", "localhost", "Host address to aergo server")
 	rootCmd.PersistentFlags().Int32VarP(&port, "port", "p", 7845, "Port number to aergo server")
+	rootCmd.PersistentFlags().StringVar(&dataDir, "keystore", "$HOME/.aergo", "Path to keystore")
```

Comment:
I wonder if it would be better if the default would be `""` for now, otherwise this immediately breaks all current uses. We can change the default in future versions (with previous announcement).

Reply:
it decided to use the default local keystore from v2.2 at team metting. so at this version release we need to announce the default change and how use node keystore like previous version

Reply:
Alright. It's usually better to announce breaking changes in advance, especially with minor releases.
So how do you use it with the old behavior? `--keystore ""`? Seems a bit counter-intuitive to have to provide an empty string. How about `--remote-keystore` as a flag which sets keystore to ""?

Reply:
added node-keystore flag